File: pythonProject/studyProject/backage/calculate.py
__all__ = ["add", "number", "Calculat"]
number = 100
name = "calculat"


def add(*args):
    if len(args) > 1:
        sum = 0
        for i in args:
            sum += i
        return sum
    else:
        print("相加至少含有两个参数")


def miuns(*args):
    if len(args) > 1:
        m = 0
        for i in args:
            m += i
        return m
    else:
        print("相减至少含有两个参数")


def multiply(*args):
    pass


def divide(*args):
    pass


class Calculat:
    def __init__(self, num):
        self.num = num

    def test(self):
        print("正在使用calculat进行运算")

    @classmethod
    def test1(cls):
        print("calculate中的类方法")


def test():
    print("我是测试")

print("__name__:----->",__name__)
if __name__ == '__main__':
    # print(__name__) # __name__ = __main__
    test()

File: src/P2st/Calculate.java
package P2st;

public class Calculate {
    // 加法定义
    public static int add(int a, int b){
        return a + b;
    }
    // 乘法定义
    public static int multiply(int a, int b){
        return a * b;
    }
    // 方法调用
    public static void main(String[] args) {
        System.out.println(add(1, 2));
        System.out.println(multiply(3, 4));
    }
}
